src/ILLink.Tasks/LinkTask.cs

```diff
@@ -190,6 +222,35 @@ protected override string GenerateResponseFileCommands ()
 				}
 			}
 
+			if (LinkSymbols)
+				args.AppendLine ("-b");
+
+			if (DefaultAction != null)
```

Comment:
I think this should be always explicitly set

Reply:
I plan to always pass in a DefaultAction from the SDK. I don't want to mark it `Required` because that seems like an unnecessary breaking change, and I think it would be good for the task defaults to match the command-line defaults (which are `UserAction`: link, and `CoreAction`: skip).

Reply:
I don't see a lot of value in setting defaults to values which everyone will have to override.

Reply:
Does it make sense for the task defaults to be different than the command-line defaults? (If everyone has to override them, could we change the command-line defaults to something reasonable?)

Reply:
it'd make sense to me to change the defaults for illink build

Reply:
Doing this in https://github.com/mono/linker/pull/1076